src/libYARP_sig/include/yarp/sig/Image.h

```diff
@@ -35,6 +36,38 @@ namespace yarp {
     }
 }
 
+// the image types partially reflect the IPL image types.
```

Comment:
Uhm, maybe inside a namespace would be better

Reply:
if (!restoreObject()) {
        unreferenceStorage();
    }

Reply:
I think this was already outside of the namespace and it was just moved above, but @aerydna please confirm this...

Reply:
i confirm